app/main.py

```diff
@@ -1 +1,44 @@
-# write your code here
+import os
+
+from app.errors import (InvalidCommandError,
+                        InvalidFileError,
+                        InvalidFileCountError,
+                        InvalidPathError)
+
+
+def move_file(command: str) -> int:
+    args_list = command.split()
+    dirs_list = []
+
+    if len(args_list) != 3:
+        raise InvalidFileCountError("Invalid file count")
+    elif args_list[0] != "mv":
+        raise InvalidCommandError("Invalid command")
+    elif args_list[1] == args_list[2]:
+        raise InvalidPathError("Invalid path for moving file")
+    elif not args_list[1].split("/")[-1]:
+        raise InvalidFileError("Invalid file name")
+    else:
+        dirs_list.extend(args_list[2].split("/"))
+        if dirs_list[-1]:
+            new_file_name = dirs_list[-1]
+            del dirs_list[-1]
+        else:
+            new_file_name = args_list[1].split("/")[-1]
```

Comment:
Simplify your solution using `os.path.dirname() ` function. It returns dir name from the path